akka-http-jsoniter-scala/src/main/scala/akka/util/FastByteString.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015 Heiko Seeberger
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package akka.util
+
+import akka.util.ByteString.ByteString1C
+
+object FastByteString {
+  def apply(bytes: Array[Byte]): ByteString =
+    if (bytes.length == 0) ByteString.empty
+    else ByteString1C(bytes)
+}
```

Comment:
is this proven to be needed?
If so, could be contributed to ByteString direclty. note we have `fromArrayUnsafe` there